src/smclient/smclient.cpp

```diff
@@ -618,6 +630,8 @@ Error SMClient::SetupChannel()
 
 Error SMClient::ReleaseChannel()
 {
+    mCertificateChanged = false;
```

Comment:
Release channel is called unlocked. Move setting `mCertificateChanged` under lock.